modules/oe_content_publication/config/install/field.field.node.oe_publication.oe_summary.yml

```diff
@@ -13,7 +13,7 @@ bundle: oe_publication
 label: Introduction
 description: 'A short text that will be displayed in the blue header, below the page title. This should be a brief summary of the content on the page that tells the user what information they will find on this page.'
 required: false
-translatable: true
+translatable: false
```

Comment:
Let's not update the field translatablility for existing fields, we have a dedicated issue for that: EWPP-169.

Reply:
This is reset.